hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/overviewCard/overviewSimpleCard.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Card, Col, Row } from 'antd';
+import { Link } from 'react-router-dom';
+import { ClusterOutlined, ContainerOutlined, DatabaseOutlined, DeleteOutlined, DeploymentUnitOutlined, FileTextOutlined, FolderOpenOutlined, InboxOutlined, QuestionCircleOutlined } from '@ant-design/icons';
+
+
+// ------------- Types -------------- //
+type IconOptions = {
+  [key: string]: React.ReactElement
+}
+
+type OverviewCardProps = {
+  icon: string;
+  data: number | React.ReactElement;
+  title: string;
+  hoverable?: boolean;
+  loading?: boolean;
+  linkToUrl?: string;
+}
+
```

Comment:
Since the styles are smaller in scale, having it inline or as a constant will have the same performance as the styles are a part of JSX which is required to be parsed in either scenario.

To improve performance we can shift to an external stylesheet, but even that would only improve negligible performance, hence having inline styles in this scenario is easier to read.

Reply:
Also for the Icons we are referencing the styles that are passed as props, hence we are defining it in the component.
Refactoring it in a separate constant would mean we cannot pass the styles, or if we want to we would need to return a component and pass the props for style twice:
`Card -> Icon Selector -> Actual icon constant`

This will decrease readability and also decrease performance slightly.

Reply:
Addressed the style changes in [8c54c99](https://github.com/apache/ozone/pull/7017/commits/8c54c99a2590b11867a6b7b4e7743fb8408360e6)